lib/bolt/pal.rb

```diff
@@ -654,17 +667,23 @@ def lookup(key, targets, inventory, executor, _concurrency)
 
         trusted = Puppet::Context::TrustedInformation.local(node).to_h
 
+        # Separate environment configuration from interpolation data the same
+        # way we do when compiling Puppet catalogs.
         env_conf = {
           modulepath: @modulepath.full_modulepath,
-          facts:      target.facts,
-          variables:  target.vars
+          facts:      target.facts
+        }
+
+        interpolations = {
+          variables: plan_vars,
```

Comment:
```suggestion
          variables:        plan_vars,
```

Hash above has values aligned.